### DISCUSSION TITLE
Version 9.10.0 looks to be half released

### DISCUSSION THREAD

**Discussion by buckett** [instructure/instructure-ui#1802] [General]
There's a commit for version 9.10.0 in github: https://github.com/instructure/instructure-ui/commit/dbb208d098b66b79183046da3e39e85439aa9dc2
There's a build in npmjs: https://www.npmjs.com/package/@instructure/ui/v/9.10.0

But there isn't a release listed in GitHub releases: https://github.com/instructure/instructure-ui/releases
And there isn't any documentation published on: https://instructure.design/v9/#CHANGELOG which lists 9.9.0 as the latest.

@HerrTopi / @matyasf - pinging you as it looks like you made the commit.

**Comment by matyasf** [comment]
oh thanks for the notice, we will fix it asap

**Reply by buckett** [reply]
Wasn't a big issue, just slightly confusing, thanks for sorting it out.
